audformat/core/index.py

```diff
@@ -156,8 +159,14 @@ def segmented_index(
             "'starts', and 'ends' differ in size",
         )
 
+    def convert_to_timedelta(times):
```

Comment:
I would move this outside of the function, as we do for the `to_array()` helper.

Reply:
Renamed it to `to_timedelta()` and made it a helper function.